irohad/main/irohad.cpp

```diff
@@ -27,16 +27,24 @@
 #include "logger/logger.hpp"
 
 /**
+ * Gflag valigator.
  * Validator for the configuration file path input argument.
  * Path is considered to be valid if it is not empty.
+ * @param flag_name - flag name
```

Comment:
flag_name - flag name? looks a bit useless

Reply:
Yes. It is standard gflag validator signature(falg_name and flag_value). In the following function flag_name is always be 'config'

Reply:
I mean the comment itself, it's not descriptive at all